### DISCUSSION TITLE
Before You Post: Welcome to The-Cessna-172-Project-V3 Discussions!

### DISCUSSION THREAD

**Discussion by CaptainBobSim** [CaptainBobSim/The-Cessna-172-Project-V3#9] [General]
## 👋 Welcome!

We hope you are enjoying the C172 Project!
If you have any issues with it please create an Issue instead so we can keep track of it easier.

We love to hear from you so fire away!
  
We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by geologistluna** [comment]
Let me start.  I am a pilot and I absolutely love to fly, however, I also love technology and love building absolutely anything.  I've been 3D printing flyable RC airplanes for many years, and love the idea of being able to 3D print my own cockpit (albeit a bit truncated).  I've had the opportunity to be in full motion 6 axis military simulators (EA-6B at NAS Whidbey island) and all though I will NOT be building a 3 story flight sim, I have been building an IFR sim that will allow me to work on my approaches while also working on other non flying related pursuits (geochemistry Ph.D).  

What will I do?  Since I teach at a major university, I will work on getting my full license back with F360, or a similar CAD/3D printer space program, and plan on trying to work on instruments like the Attitude indicator, and Altimeter.  These are sufficiently complex that few have created a properly working system.  As an example, no 3D printed instrument has an error flag for the attitude indicator to indicate that the gyros are spinning too slow, we should work on that.  What about flags for the heading indicator or altimeter?   Again, not impossible to do, just very difficult with FDM.  I will build upon those.   Realism is what I want. I want to be sure that if an instrument goes down, it goes down in the appropriate manner. 

I love the challenge of building a fully functioning 3D printed Cessna 172 cockpit.

**Reply by CaptainBobSim** [reply]
That's awesome! 

If you are looking for things to work on (and no pressure!):
Heading Indicator: All done! You can add anything you'd like! I just updated the GitHub to the latest version
Altimeter: Almost done - would like to finish some tweaks (and maybe post video?) before handing it over
Attitude Indicator: I've gotten a rough proof of concept. If you want to you can take the reigns on this, Ideally, it'd be done within a month or two because it is the next video after Altimeter. 
I love the challenge too! That's what makes this hobby so thrilling!

**Reply by CaptainBobSim** [reply]
Also things in the No Status/Designing Category are likely up for grabs - just let me know! https://github.com/users/CaptainBobSim/projects/2
